src/command/dev/mcp/binary.rs

```diff
@@ -78,6 +77,35 @@ pub struct RunMcpServerBinary<Spawn: Send> {
     env: HashMap<String, String>,
 }
 
+impl<Spawn: Send> RunMcpServerBinary<Spawn> {
+    // Gather the rover-specific configuration options into environment variables
+    // understood by the MCP server.
+    // TODO: Magic strings are not fun to debug later.
+    fn opts_into_env(self) -> HashMap<String, String> {
+        let overlayed = HashMap::from([
```

Comment:
nit: English isn't my first language so I'm not confident but I think this is correct? 😅

```suggestion
        let overlaid = HashMap::from([
```

Reply:
Whoops, good catch!